bin/node-template/pallets/template/src/lib.rs

```diff
@@ -42,7 +43,7 @@ pub mod pallet {
 	// https://docs.substrate.io/main-docs/build/events-errors/
 	#[pallet::event]
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
-	pub enum Event<T: Config> {
+	pub enum PalletEvent<T: Config> {
```

Comment:
@KiChjang some bikeshdding: I think this is rooting back to your initial suggestion [here](https://github.com/paritytech/substrate/issues/11736#issuecomment-1163169701) and having seen the outcome, I want to suggest keeping this enum called `Event` in the pallet and _ONLY_ prefixed with `Runtime` (i.e. `RuntimeEvet`) the outer enum. The reason being that within this code, we already know that this is inside a pallet, there's no need to emphasize again that this is the `Event` of a `Pallet`. For example, `pallet_balances::Event` makes a lot more sense to me than `pallet_balances::PalletEvent`. 

Moreover, if we call this `PalletEvent`, then one would ask why is `Event` called `PalletEvent` and `Origin` is not called `PalletOrigin`? 

Regardless of `Event` or `PalletEvent`, the current outcome is not consistent. All enums and structs inside a pallet should be prefixed with `Pallet`, not not prefixed at all and we should certainly chose one and stick to it (and clearly my preference is to ditch `Pallet` prefix). 

If you agree, it would be easy enough for @Szegoo to apply a blanket `PalletEvent` -> `Event` replacement script here.

Reply:
This would also lessen the diff size here.

Reply:
I had the same thought too, but I guess we'd still want some sort of differentiation between `Event` and `RuntimeEvent`... perhaps what we can do is keep it as `Event`, but create a type alias `type PalletEvent = Event` on the pallet itself during macro expansion?

Reply:
@KiChjang But if we make a type alias, when would we actually want to use the `Event` and when the `PalletEvent`, I assume that you meant to use the `Event` inside the pallet itself, and use `PalletEvent` outside the pallet. Wouldn't this be a bit confusing?

Reply:
I think the fewer keywords, the better, therefore not a fan of `PalletEvent` alias.

Reply:
Ok, so what is the final agreement then, should I change the `PalletEvent` back to `Event`?

Reply:
> I think the fewer keywords, the better, therefore not a fan of `PalletEvent` alias.

This is then saying that there's no need to differentiate between a runtime event and a pallet event? It is not a given that we always import the pallet event type by prepending `my_pallet::Event`. If we see a free-standing `Event` that's not part of a pallet module, do we then assume it's from a pallet event?

Reply:
> Ok, so what is the final agreement then, should I change the `PalletEvent` back to `Event`?

This should happen regardless. What we are discussing right now is another matter.

Reply:
> If we see a free-standing Event that's not part of a pallet module, do we then assume it's from a pallet event?

yeah, exactly.